src/components/AddBookForm.jsx

```diff
@@ -1,12 +1,28 @@
 import React from 'react';
+import { useDispatch } from 'react-redux';
+import { addBook } from '../redux/books/books';
 
-const AddBookForm = () => (
-  <form>
-    <h2>Add New Book</h2>
-    <input type="text" name="bookname" placeholder="Book title" />
-    <input type="dropdown" name="category" placeholder="Select category" />
-    <input type="submit" value="Add Book" />
-  </form>
-);
+const AddBookForm = () => {
+  const dispatch = useDispatch();
+
+  const handleSubmit = (event) => {
+    event.preventDefault();
+    dispatch(addBook({
+      id: Date.now(),
+      title: event.target[0].value,
+      author: event.target[1].value,
+    }));
+    event.target.reset();
+  };
+
+  return (
+    <form onSubmit={(event) => { handleSubmit(event); }}>
```

Comment:
- [ ] You did a good job here, but in the project requirements the submission should be done through a button, so please create a button that creates new books. You can accomplish this by using a local state to track changes.